integrations/Refiner/browser.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable */
+import logger from "../../utils/logUtil";
+import { NAME } from "./constants";
+import { replaceUserTraits, replaceAccountTraits } from "./utils";
+
+class Refiner {
+  constructor(config) {
+    this.name = NAME;
+    this.apiKey = config.webClientApiKey;
+    this.userAttributesMapping = config.userAttributesMapping;
+    this.accountAttributesMapping = config.accountAttributesMapping;
+  }
+
+  loadScript() {
+    window._refinerQueue = window._refinerQueue || [];
+    this._refiner = function () {
+      window._refinerQueue.push(arguments);
+    };
+    (function () {
+      var a = document.createElement("script");
+      a.type = "text/javascript";
+      a.async = !0;
```

Comment:
Add the below line:
`import { LOAD_ORIGIN } from '../ScriptLoader';`
`a.setAttribute('data-loader', LOAD_ORIGIN);`